pybamm/input/parameters/lithium-ion/anodes/graphite_Ai2020/parameters.csv

```diff
@@ -46,3 +46,8 @@ Negative electrode Paris' law constant b,1.12,,
 Negative electrode Paris' law constant m,2.2,,
 Negative electrode cracking rate,[function]graphite_cracking_rate_Ai2020,,
 Negative electrode activation energy for cracking rate [J.mol-1],0,,
+,,,
+# Loss of active materials (LAM) model,,,
+Negative electrode LAM constant beta,0,,
+Negative electrode LAM constant m,2,,
```

Comment:
Can we use more informative names for `beta` and `m`?

Reply:
yes, will use propotional term and exponential term instead